Makefile

```diff
@@ -1,33 +1,14 @@
-VERSION=0.1
+default: build
 
-GOFILES = $(shell find . -name '*.go' -not -path './vendor/*')
+build: build/linux/amd64/circleci build/darwin/amd64/circleci
 
-OS ?= $(shell uname)
-
-CLIPATH=github.com/CircleCI-Public/circleci-cli
-
-EXECUTABLE=circleci-cli
-BUILD_DIR=build
-
-.PHONY: build
-build:
-ifeq ($(OS), Darwin)
-	GOOS=darwin GOARCH=amd64 go build -o $(BUILD_DIR)/darwin/amd64/$(EXECUTABLE)
-else ifeq ($(OS), Linux)
-	GOOS=linux GOARCH=amd64 go build -o $(BUILD_DIR)/linux/amd64/$(EXECUTABLE)
-endif
-
-.PHONY: build/*
-$(BUILD_DIR)/%/amd64/$(EXECUTABLE): $(GOFILES)
-	GOOS=$* GOARCH=amd64 CGO_ENABLED=0 go build $(LDFLAGS) -o $(BUILD_DIR)/$*/amd64/$(EXECUTABLE) .
-
-.PHONY: build-all
-build-all: $(BUILD_DIR)/darwin/amd64/$(EXECUTABLE) $(BUILD_DIR)/linux/amd64/$(EXECUTABLE)
+build/%/amd64/circleci: always
+	GOOS=$* GOARCH=amd64 go build -v -o $@ .
```

Comment:
Am I understanding this correctly, will build both linux and mac binaries every time?

For some convenience, we wanted `make` to just build the current OS's binary to save some time.